behat.yml.dist

```diff
@@ -46,7 +46,7 @@ default:
         FriendsOfBehat\SymfonyExtension:
             bootstrap: tests/Application/config/bootstrap.php
             kernel:
-                class: Tests\Webgriffe\SyliusNexiPlugin\Application\Kernel
```

Comment:
Why this change? The namespace was correct:
https://github.com/webgriffe/SyliusNexiPlugin/blob/main/tests/Application/Kernel.php#L5